app/helpers/content_helper.rb

```diff
@@ -1,6 +1,8 @@
 module ContentHelper
   def render_content_page(page_name)
-    @converted_markdown = GovukMarkdown.render(File.read("app/views/content/#{page_name}.md")).html_safe
+    raw_content = File.read("app/views/content/#{page_name}.md")
+    content_with_erb_tags_replaced = ERB.new(raw_content).result
```

Comment:
How about

```suggestion
    content_with_erb_tags_replaced = ERB.new(raw_content).result(binding)
```

This lets us use `t` directly in the template without needing to reference `I18n`?

Reply:
Ah, this breaks `t!` but plain `t` works. I wonder what's going on there. It's good to raise on missing translations (as we do in `application.rb`) but that doesn't seem to affect this manual ERB rendering. Anyway, would it be nice not to have to remember to call `I18n.t!` instead of `I18n.t`, at risk of letting through "Translation missing..." strings into rendered templates?

Reply:
OK, how about this?

```suggestion
    content_with_erb_tags_replaced = ApplicationController.renderer.render(inline: raw_content)
```

and then plain `t` in the template

From https://api.rubyonrails.org/classes/ActionController/Renderer.html

Reply:
@duncanjbrown missed your last message before pushing a new commit! Let's go with your one, much clearer.